core/include/traccc/fitting/kalman_fitting_algorithm.hpp

```diff
@@ -29,11 +29,13 @@ namespace traccc::host {
 class kalman_fitting_algorithm
     : public algorithm<track_state_container_types::host(
           const default_detector::host&,
-          const detray::bfield::const_field_t::view_t&,
+          const detray::bfield::const_field_t<
+              default_detector::host::scalar_type>::view_t&,
           const track_candidate_container_types::const_view&)>,
       public algorithm<track_state_container_types::host(
           const telescope_detector::host&,
-          const detray::bfield::const_field_t::view_t&,
+          const detray::bfield::const_field_t<
+              default_detector::host::scalar_type>::view_t&,
```

Comment:
```suggestion
              telescope_detector::host::scalar_type>::view_t&,
```

I marked this one after all. 😛 Some of the mistakes were made in this direction as it seems.